OCR/frontend/src/components/ImageAnnotator.tsx

```diff
@@ -35,7 +35,7 @@ export const MultiImageAnnotator: FC<MultiImageAnnotatorProps> = ({ images, cate
         const field = fields.find(field => field.name === selectedField?.name);
         const updatedShapes = [...shapes];
         // todo fix typing but for now this is fine
-        updatedShapes[index] = [...(updatedShapes[index] || []), {...shape, field: selectedField?.name as string, color: field?.color}];
+        updatedShapes[index] = [...(updatedShapes[index] || []), {...shape, field: selectedField?.name as string, color: field?.color.slice(0,7)}];
```

Comment:
getting this error on this line
![image](https://github.com/user-attachments/assets/7826032d-79ab-43b6-b7a1-608cb5e987a6)